ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx

```diff
@@ -12,47 +12,47 @@ import { Loader } from '../../partials';
 import { InfiniteScroll } from '../../utils';
 
 const connections = {
-  "live": {
-    "0": 560,
-    "1": 300,
-    "2": 245,
-    "3": 198,
-    "4": 237,
-    "5": 564,
-    "6": 2545,
-    "7": 3253,
-    "8": 5284,
-    "9": 4573,
-    "10": 3142,
-    "11": 2573,
-    "12": 1642,
-    "13": 3573,
-    "14": 3235,
-    "15": 3463,
-    "16": 4074,
-    "17": 5036,
-    "18": 4546,
-    "19": 4055,
-    "20": 2573,
-    "21": 2024,
-    "22": 944,
-    "23": 553
-  },
-  "limit": "6.000",
-  "max": "5.603",
-  "avg": "3.740",
-  "trend": "up"
+    "live": {
```

Comment:
It's better to avoid whitespace changes in feature PRs. Easier to read if they are in separate PRs or commits.